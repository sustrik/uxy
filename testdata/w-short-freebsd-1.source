 9:44AM  up 112 days, 18:52, 1 users, load averages: 0.03, 0.10, 0.14
USER       TTY      FROM                                      LOGIN@  IDLE WHAT
sustrik    pts/0    51.154.87.41                              9:42AM     - w

