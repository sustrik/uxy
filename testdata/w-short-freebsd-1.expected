USER       TTY      FROM                                      LOGIN@  IDLE WHAT 
sustrik    pts/0    51.154.87.41                              9:42AM  -    w   
